src/pf-popover/index.html

```diff
@@ -0,0 +1,96 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>pf-popover example</title>
+    <link rel="stylesheet" href="../../dist/css/patternfly.css">
+    <link rel="stylesheet" href="../../dist/css/patternfly-additions.css">
+    <link rel="stylesheet" href="../../dist/css/patternfly-webcomponents.css">
+    <style>
+        #div {
+            margin-top: 40px;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="container">
+        <div class="page-header">
+            <h1>popovers</h1>
+        </div>
+        <hr>
+
+            <pf-popover id="popover-right" placement="right" targetSelector="#btn-right">
+                popover text on right popover text on right popover text on right popover text on right popover text on right
+            </pf-popover>
+            <button id="btn-right" type="button" class="btn btn-primary" aria-describedby="popover-right">use of public handler(toggle()) to open</button>
+
+        <div id="div">
+            <pf-popover id="popover-right" placement="right" targetSelector="#btn-right-t-c" popoverTitle="Popver on right" dismissible="true">
+                popover text on right popover text on right popover text on right popover text on right popover text on right
+            </pf-popover>
+            <button id="btn-right-t-c" type="button" class="btn btn-primary" aria-describedby="popover-right">Popover on right with title and close button</button>
+        </div>
+
+        <div id="div">
+            <pf-popover id="popover-right" placement="right" targetSelector="#btn-right-t" popoverTitle="Popver on right">
```

Comment:
I've created [#53](https://github.com/patternfly-webcomponents/patternfly-webcomponents/issues/53) to track this. I think we can clean up attribute names to be consistent across in a new PR...